### DISCUSSION TITLE
Contributor Call: 12 November 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3025] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 12 November 2022 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/nov-12-2022/5pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.
⚠There have been timezone switches lately, careful!

### Agenda
- TBD

> Notes will be added after the call.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark
- @lexander2002
- @KaerMorhan
- @joostdecock
- @bobgeorgethe3rd
- @BenJamesBen
- @SeaZeeZee
- @eriese 
- @eleonoremilbou 
- @anna-puk 
- @sorchanidhubhghaill (briefly)
- Unagi
- Cani[FLH]

✨ **Enoch**
I’m having troubles with pattern layout. Asking a permission from Joost to create a method that would allow to independently draft parts after pattern packing.
_Joost: the problem with that is that drafting a part requires drafting other part, so it will be double work trying to create each part in context._
_Enoch: I’m not trying to draft part by part._
_Joost: The way it’s done right now is the pattern is drafted, then a new part is added and it’s drafted again. It doesn’t require a hook._
Adding and drafting a pattern after packing. Drafting a part that doesn’t have any dependencies.
_Joost: Yeah, that sounds reasonable to me, this is not a bad idea._

✨ **Unagi**
I’m a developer, I would like to help with bigger sizes and women’s patterns (Bella), also maybe helping with French translation.
_Joost: We’re moving to v3. The thing that I struggle to do to this day is to explain to people what FreeSewing is. To make a landing page that will give a very clear idea on what FS is. I’m also thinking of giving users a choice of the complexity of the website that would be on a scale._
_Unagi: We need more filters for sure, especially with more and more patterns. Just tags would help._
_Joost: Yeah, we’re kind of in the limbo, when the backend will be ready, we will start working on the website._

✨ **Benjamin**
There are several bugs I’m working on right now. One of them is Bezier curves bug (extra parts/lines).
_Joost: So Benjamin discovered that this issue comes from a pack we’re using for the curve building. The question now is can we fix it with a later code._
Do we need a debug code? We probably don’t as it makes things more complicated.
We can merge the request, leave it in for now and make a mental note on fixing it later or removing it completely.

✨ **Zwaluw**
Maybe we could also do something for user-friendliness who expect a perfect pattern and don’t get it.
_Joost: Usually beginners get more good results, and experienced users do not expect the perfect pattern, maybe there’s a third group in-between though. Putting a disclaimer would be disrespecting to the work we put in as people would expect our patterns to be bad. Maybe we could add it to the "Getting started" page._
_Zwaluw: I’m in a third group kinda. I know that commercial patterns for pants don’t work for me so I expect that drafted to my measurements would be good but sometimes we have measurements that designer didn’t anticipate._
_Unagi: Maybe we could show an array of different patterns and say “look at all those different forms which are not perfect”. We could put an accent on body-positivity._
_Joost: And returning to the scale of website complexity that shows you more stuff, maybe it will also not show something (like disclaimers for more complicated levels)._
_Ben: Maybe we could make a tutorial on how to use FreeSewing._
_Joost: Best case scenario is a catchy video._
(after collective discussion) _We could use a voice-over for different languages or do subtitles._

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
